test/generators/avo_install_generator_test.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string-literal: true
+
+require 'test_helper'
+require 'generators/boring/avo/install/install_generator'
+
+class AvoInstallGeneratorTest < Rails::Generators::TestCase
+  tests Boring::Avo::InstallGenerator
+  setup :build_app
+  teardown :teardown_app
+
+  include GeneratorHelper
+
+  def destination_root
+    app_path
+  end
+
+  def test_should_install_avo_gem_successfully
```

Comment:
Let's remove "successfully" since install will either be success or fail with error, this word is just redundant due to that.